src/contentscript/overlay/root/pages/Dapplets/index.tsx

```diff
@@ -291,7 +291,12 @@ export const Dapplets: FC<DappletsProps> = (props) => {
       {isLoadingListDapplets ? (
         <TabLoader />
       ) : (
-        <div className={cn(styles.dappletsBlock, classNameBlock)}>
+        <div
+          className={cn(styles.dappletsBlock, classNameBlock, {
+            [styles.noScroll]:
+              isNoContentScript || !filteredDapplets || filteredDapplets.length === 0,
+          })}
```

Comment:
A scroll must be displayed only when container is overfilled and must be implemented using only CSS, no JavaScript. The current implementation doesn't work when number of elements > 0 and container is not overfilled:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/15713727/220515301-802fb85e-be1b-4e5f-89d4-9d4d7c837659.png">

Reply:
+